hw2/02-search.js

```diff
@@ -0,0 +1,76 @@
+// Sources: https://youtu.be/Ixt6k9aatPQ  (Highlighting in JavaScript)
+
+search.addEventListener("click", handleClick);
+const section = document.getElementById("holder");
+const elem = document.querySelector("input");
+
+function handleClick(event) {
+  handleInput(event);
+}
+
+function handleInput(event) {
+  const result = charaSearch(characters);
+  displayCharacters(result);
+}
+
+const charaSearch = (input) => {
+  let result = [];
+  input.forEach((element) => {
+    if (element.name.includes(elem.value)) {
```

Comment:
I think including toLowerCase() would be nice because it allows users to enter lower case or upper case and get the same result.